Automatic Validation ended with:
> 2025-02-15T04:23:43.6024058Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Tags] Description: Elements at indexes #1 and #2 violate uniqueness constraint. Error context: <root> Description: Failed to validate against schema associated with property name 'Tags'. ] File: Mozilla.Thunderbird.locale.en-US.yaml Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Tags] Description: Elements at indexes #1 and #2 violate uniqueness constraint. Error context: <root> Description: Failed to validate against schema associated with property name 'Tags'. ] File: Mozilla.Thunderbird.locale.es-ES.yaml
 2025-02-15T04:23:43.6036262Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Tags] Description: Elements at indexes #1 and #2 violate uniqueness constraint. Error context: <root> Description: Failed to validate against schema associated with property name 'Tags'. ] File: Mozilla.Thunderbird.locale.en-US.yaml Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Tags] Description: Elements at indexes #1 and #2 violate uniqueness constraint. Error context: <root> Description: Failed to validate against schema associated with property name 'Tags'. ] File: Mozilla.Thunderbird.locale.es-ES.yaml

(Automated response - build 980.)

--- a/manifests/m/Mozilla/Thunderbird/78.4.0/Mozilla.Thunderbird.locale.en-US.yaml
+++ b/manifests/m/Mozilla/Thunderbird/78.4.0/Mozilla.Thunderbird.locale.en-US.yaml
@@ -21,7 +21,6 @@ Moniker: thunderbird
 Tags:
 - e-mail
 - email
-- email
 - email-client
 - thunderbird
 # ReleaseNotes:

--- a/manifests/m/Mozilla/Thunderbird/78.4.0/Mozilla.Thunderbird.locale.es-ES.yaml
+++ b/manifests/m/Mozilla/Thunderbird/78.4.0/Mozilla.Thunderbird.locale.es-ES.yaml
@@ -21,7 +21,6 @@ ShortDescription: Thunderbird es un gestor de correo gratuito fácil de configur
 Tags:
 - e-mail
 - email
-- email
 - email-client
 - thunderbird
 # ReleaseNotes:

